.github/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ## 0.8.0
 
+1. [EFB] Fix Metar parser error - @brainshot (brainshot)
```

Comment:
Please move this to the end of the list.

Reply:
done